bigframes/core/nodes.py

```diff
@@ -341,20 +363,38 @@ def order_ambiguous(self) -> bool:
     def explicitly_ordered(self) -> bool:
         return True
 
-    def transform_children(
-        self, t: Callable[[BigFrameNode], BigFrameNode]
-    ) -> BigFrameNode:
-        return self
+    @property
+    def row_count(self) -> typing.Optional[int]:
+        return self.n_rows
 
 
-## Put ordering in here or just add order_by node above?
 @dataclass(frozen=True)
-class ReadTableNode(BigFrameNode):
+class GbqTable:
```

Comment:
This is so we can get something hashable? So we can make sure we only have the fields we care about? A docstring with the purpose would be helpful here.